kompose.spec

```diff
@@ -967,6 +973,18 @@ export LDFLAGS=%{ldflags}
 %doc RELEASE.md CHANGELOG.md README.md CONTRIBUTING.md ROADMAP.md code-of-conduct.md
 %{_bindir}/kompose
 
+%post
+if [ -z ${BASH+x} ];
```

Comment:
hmm. what does `${BASH+x}` give you? looks like `x` if $BASH is defined and `` if not. What is the benefit of doing this and not `-z "$BASH"` ?